docs/user/get_started.md

```diff
@@ -0,0 +1,233 @@
+| Behavioral Cloning     | PyTorch             |
+
+They are organized in the [github repository](https://github.com/rlworkgroup/garage/tree/master/src/garage) as:
+
+```bash
+└── src
+    └── garage
+        ├── envs
+        ├── experiment
+        ├── misc
+        ├── [np](https://github.com/rlworkgroup/garage/tree/master/src/garage/np)
+        ├── plotter
+        ├── replay_buffer
+        ├── sampler
+        ├── [tf](https://github.com/rlworkgroup/garage/tree/master/src/garage/tf)
+        └── [torch](https://github.com/rlworkgroup/garage/tree/master/src/garage/torch)
+```
+
+Note: clickable links represents the directory of algorithms.
+
+A simple pytorch example to import `TRPO` algorithm, as well as, the policy `GaussianMLPPolicy` and value function `GaussianMLPValueFunction` in garage is shown below:
+```py
+import gym
+import torch
+
+from garage.envs import GarageEnv, normalize
+from garage.torch.algos import TRPO as PyTorch_TRPO
+from garage.torch.policies import GaussianMLPPolicy as PyTorch_GMP
+from garage.torch.value_functions import GaussianMLPValueFunction
+
+def trpo_garage_pytorch():
+
+    env = GarageEnv(normalize(gym.make(env_id))) # specify env_id
+
+    policy = PyTorch_GMP(env.spec,
+                         hidden_sizes= [32, 32],
+                         hidden_nonlinearity=torch.tanh,
+                         output_nonlinearity=None)
+
+    value_function = GaussianMLPValueFunction(env_spec=env.spec,
+                                              hidden_sizes=(32, 32),
+                                              hidden_nonlinearity=torch.tanh,
+                                              output_nonlinearity=None)
+
+    algo = PyTorch_TRPO(
+        env_spec=env.spec,
+        policy=policy,
+        value_function=value_function,
+        max_episode_length=100,
+        discount=0.99,
+        gae_lambda=0.97)
+
+```
+
+The full code can be found [here](https://github.com/rlworkgroup/garage/blob/master/benchmarks/src/garage_benchmarks/experiments/algos/trpo_garage_pytorch.py).
+
+To know more about implementing new algorithms, see [this guide](https://garage.readthedocs.io/en/latest/user/implement_algo.html)
+
+### Runnning Experiments
+
+In garage, experiments are run using the "experiment launcher" `wrap_experiment`, a decorated Python function, which can be imported directly from the garage package.
+
+```py
+from garage import wrap_experiment
+```
+
+Moreover, objects, such as `LocalRunner`, `environment`, `policy` e.t.c are commonly used when constructing experiments in garage.
+
+```py
+import gym
+import torch
+
+from garage import wrap_experiment
+from garage.envs import GarageEnv, normalize
+from garage.experiment import deterministic, LocalRunner
+from garage.torch.algos import TRPO as PyTorch_TRPO
+from garage.torch.policies import GaussianMLPPolicy as PyTorch_GMP
+from garage.torch.value_functions import GaussianMLPValueFunction
+
+hyper_parameters = {
```

Comment:
One of the key points of `wrap_experiment` is to avoid this hyper-parameter pattern, so it's weird to use it in the documentation here.